MotorControl/MotorControl/Resources/Tremor.json

```diff
@@ -7,7 +7,7 @@
              {
                 "identifier":"introduction",
                 "type":"overview",
-                "title":"Phone Hold",
+                "title":"Tremor Test",
```

Comment:
Figma says

mPower: "Tremor Test"
pkYou: "Resting tremor"

Note: The capitalization of the title is inconsistent across both Figma designs. In all cases, I have gone with using book title capitalization.

![Screen Shot 2019-10-23 at 10 58 34 AM](https://user-images.githubusercontent.com/5649217/67420924-5ca02c80-f584-11e9-8882-32d4b5261ac5.png)
![Screen Shot 2019-10-23 at 11 01 34 AM](https://user-images.githubusercontent.com/5649217/67421012-835e6300-f584-11e9-95c1-71e596a17ad5.png)